core/client/fs/src/main/java/alluxio/client/file/FileOutStream.java

```diff
@@ -315,7 +315,7 @@ private void handleCacheWriteException(Exception e) throws IOException {
   @ThreadSafe
   private static final class Metrics {
     private static final Counter BYTES_WRITTEN_UFS =
-        MetricsSystem.counter(WorkerMetrics.BYTES_WRITTEN_UFS);
+        MetricsSystem.counter(ClientMetrics.BYTES_WRITTEN_UFS);
```

Comment:
Note this is unrelated to the PR. However, this is require to make the unit test effective. Although we incorrectly track metrics from the worker, we also have client metrics in our test environment which remedies the problem.

A separate question is whether or not we want to keep client metrics for operations which will be tracked by the server (this can be useful if the client itself is reporting to an external metrics server) and how to coordinate between the client and server to ensure metrics are not double counted.

Reply:
If my understanding I think `BYTES_WRITTEN_UFS` is only applicable on the client if the user is using an implementation of `LocalUnderFileSystem`(i.e. local development or NFS)? Otherwise, the client will always need to send the bytes through the worker for writing to the UFS.

For this test, if you set ` alluxio.user.short.circuit.enabled=false` can we get away without this change? 

For the most part I feel `BYTES_WRITTEN_UFS` doesn't make sense to keep track of as a client metric.

Reply:
This tracks the number of bytes we write if we are using a sync persist mode. It will duplicate the metric (but without tags, which is why the test will pass with or without the actual fix).